processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableContainerGenerator.java

```diff
@@ -69,18 +75,21 @@ private static void addListMultimapWithGetters(
         FieldSpec.builder(keyMapType, fieldName, Modifier.PRIVATE)
             .initializer("$T.create()", ParameterizedTypeName.get(ArrayListMultimap.class))
             .build());
+    String sortedBy = sequenceField != null ? " sorted by " + sequenceField.name() : "";
```

Comment:
Nit: consider using FieldNameConverter.gtfsColumnName(sequenceField.name()) to get a_name_with_underscores_like_in_gtfs.